wp-content/themes/core/theme.json

```diff
@@ -202,6 +202,26 @@
 					"shadow": "0px 0px 1px rgba(48, 49, 51, 0.05), 0px 8px 50px rgba(48, 49, 51, 0.1)"
 				}
 			]
+		},
+		"blocks": {
+			"core/button": {
+				"border": {
+					"color": false,
+					"radius": false,
+					"style": false,
+					"width": false
+				},
+				"typography": {
+					"fontFamilies": [],
+					"fontSizes": [],
+					"textTransform": false
+				}
+			},
+			"core/group": {
+				"color": {
+					"background": true
```

Comment:
Added this so we can test elements/blocks on a background easier, like [this](http://p.tri.be/i/kbORAK)